corehq/apps/export/templates/export/customize_export.html

```diff
@@ -391,6 +391,16 @@
                 self.save.state = ko.observable('save');
 
                 self.save_no_preview = function() {
+                    var exportType = self.export_type();
+                    exportType = exportType.charAt(0).toUpperCase() + exportType.slice(1);
+                    var action = "Regular";
+                    if (self.presave()) {
+                        action = "Saved";
```

Comment:
also nice if there were a list of actions as defined constants so it's less error prone and we're able to see all the actions in one place